src/infra/config/dex/paraswap/file.rs

```diff
@@ -25,6 +25,9 @@ struct Config {
 
     /// Which partner to identify as to the paraswap API.
     pub partner: String,
+
+    /// Which chain the solver is serving.
+    pub chain_id: u64,
```

Comment:
This could be deduced from the `node-url` of the base configuration but that would make testing extremely inconvenient (basically requires to spin up a server only to return `chainId: 1`).